phpunit.xml.dist

```diff
@@ -2,11 +2,8 @@
 <phpunit bootstrap="./tests/bootstrap.php" colors="true">
 
     <testsuites>
-        <testsuite name="unit">
-            <directory suffix="Test.php">./src/Sulu/Component/*/Tests/Unit</directory>
-        </testsuite>
-        <testsuite name="functional">
-            <directory suffix="Test.php">./src/Sulu/Component/*/Tests/Functional</directory>
+        <testsuite name="components">
```

Comment:
Ok, but I think that's something open to discuss. I think there should not be any functional tests in the component, because the components are only fully integrated in the bundles, and that's where functional testing should be happening.